routes/extensionRequests.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const router = express.Router();
+const extensionRequests = require("../controllers/extensionRequests");
+const authenticate = require("../middlewares/authenticate");
+const authorizeRoles = require("../middlewares/authorizeRoles");
+const { SUPERUSER, APPOWNER } = require("../constants/roles");
+const {
+  createExtensionRequest,
+  updateExtensionRequest,
+  updateExtensionRequestStatus,
+} = require("../middlewares/validators/extensionRequests");
+
+router.post("/", authenticate, createExtensionRequest, extensionRequests.createTaskExtensionRequest);
```

Comment:
the controller checks if the user is a superuser, what about a normal user? Is it a superuser-only feature, couldn't we use authorize roles method?

Reply:
even a normal user should be able to create an extension request right?